projects/netgrif-components/package.json

```diff
@@ -60,6 +60,5 @@
         "natural-orderby": "~2.0.3",
         "rxjs": "~6.6.0",
         "zone.js": "~0.11.4"
-    },
-    "schematics": "schematics/collection.json"
```

Comment:
the "./" path prefix of the schematics collection was removed in the core library package.json (on line 7) but not here, why?

Reply:
it was fixed by @renczesstefan